cleanlab/rank.py

```diff
@@ -427,6 +429,133 @@ def get_label_quality_ensemble_scores(
     return label_quality_scores
 
 
+def get_worst_class(labels, pred_probs):
+    ranked_noisy_classes = rank_classes_by_label_quality(labels, pred_probs).query(
+        "`Label Quality Score` < 1.0"
+    )
+    return ranked_noisy_classes["Class Index"][0] if len(ranked_noisy_classes) > 0 else np.nan
+
+
+def get_label_quality_scores_multiannotator(
+    labels_multiannotator: pd.DataFrame,
+    pred_probs: np.array,
+    *,
+    consensus_method: Union[str, List[str]] = "majority",
+    return_annotator_stats: bool = False,
```

Comment:
missing documentation for this argument

Reply:
Returned annotator_stats DF should also be documented

Reply:
Let's add another arg here:
`verbose = True`

If verbose: then the full `annotator_stats` DF is printed out during the execution of this function.

Reply:
done!